libs/wire-subsystems/test/unit/Wire/UserSubsystem/InterpreterSpec.hs

```diff
@@ -32,7 +33,7 @@ import Wire.API.User hiding (DeleteUser)
 import Wire.MiniBackend
 import Wire.StoredUser
 import Wire.UserSubsystem
-import Wire.UserSubsystem.Interpreter -- TODO: only import UserSubsystemConfig(..)
```

Comment:
please don't remove this.  i'll add it again and add my name, maybe that helps.  :)